server/src/converter.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const path = require('path');
+const puppeteer = require('puppeteer');
+const handlebars = require('handlebars');
+
+async function createPDF(data) {
+  const templateHtml = fs.readFileSync(
+    path.join(process.cwd(), 'pdf-template.html'),
+    'utf8'
+  );
+  const template = handlebars.compile(templateHtml);
+  const html = template(data);
+
+  const pdfPath = path.join('pdf', `${data.name}.pdf`);
+
+  const options = {
+    format: 'A4',
+    headerTemplate: '<p></p>',
+    footerTemplate: '<p></p>',
+    displayHeaderFooter: false,
+    margin: {
+      top: '40px',
+      bottom: '100px',
+    },
+    printBackground: true,
+    path: pdfPath,
+  };
+
+  try {
+    const browser = await puppeteer.launch({
+      args: ['--no-sandbox'],
+      headless: true,
+    });
+
+    const page = await browser.newPage();
+
+    await page.setContent(`${html}`, {
+      waitUntil: 'networkidle0',
+    });
+
+    await page.pdf(options);
+    await browser.close();
+  } catch (err) {
+    console.log(err);
```

Comment:
Use the appLogger for logging things - that way we centralize all of the logging in one file

`src/logging/appLogger.js`

Reply:
done. this is the module caused the import error initially but managed to fix